### PR TITLE
GatedTabTransformer Implementation.

### PR DESCRIPTION
**Pull request by radi-cho**
Implementation of [GatedTabTransformer](https://arxiv.org/abs/2201.00199). Relies on #362. Requested by @radi-cho in #354 and approved by @oguiza.

### CODE REVIEW

**Comment by oguiza** [comment]
Hi @radi-cho,
Thanks a lot for creating the 2 PRs. I'm pretty busy at the moment, but I'll try to find some time to review both papers and the PRs later this week and will come back to you with comments.

**Comment by radi-cho** [comment]
@oguiza Thank you. We'll be having a presentation about our paper on Saturday and I was hoping that we can finish this by then. I will be regularly online until Friday night, so I will be thankful if you can send us some feedback before that for us to fix and have the PRs ready to be merged.

**Comment by radi-cho** [comment]
@oguiza Any updates on this?

**Comment by oguiza** [comment]
> @oguiza Any updates on this?

I'm sorry @radi-cho but I'm really busy at this time. I believe I'll be able to review the papers and your PRs next week.
Thanks for your patience!

File: nbs/121b_models.GatedTabTransformer.ipynb [hunk 1]
```diff
@@ -0,0 +1,151 @@
+{
```
Comment (oguiza) [inline nbs/121b_models.GatedTabTransformer.ipynb:1]: <p>Hi @radi-cho,</p><p>This implementation looks good to me. </p><p>BTW, are you the author of the arxiv paper?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/timeseriesAI/tsai/pull/365/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Comment (radi-cho) [reply nbs/121b_models.GatedTabTransformer.ipynb:1]: Thank you!
Yes, I am the author.

**Comment by radi-cho** [comment]
@oguiza Synced with #362. Git conflicts have been resolved. Everything has been tested here as well.

**Comment by radi-cho** [comment]
@oguiza Can we have these pull requests (#365 and #362) merged?
Please confirm they are eligible and I will solve the conflicts again so they can be merged more quickly.

**Pull request closed without merging**